### DISCUSSION TITLE
Rotating Cartesian axes

### DISCUSSION THREAD

**Discussion by psads-git** [cetz-package/cetz#594] [Q&A] [answered]
I have created the following plot in `tikz`. I was also able to reproduce the same plot with `cetz`, by using calculations with `sin` and `cos` to rotate the axes. I am wondering whether we can use `ortho` to simplify the code to rotate the axes.

![image](https://github.com/cetz-package/cetz/assets/75945439/90012f7b-c4fe-48c9-96db-1e887fb586b9)

**Comment by johannes-wolf** [comment, marked as answer]
You can just use `rotate(<angle>)` to rotate around the z-axis (or any axis if you specify the axis name as named parameter).
Example:
```typst
#import "@preview/cetz:0.2.2": canvas, draw
#set page(width: auto, height: auto, margin: .5cm)

#canvas({
  import draw: *

  set-style(mark: (length: .1, width: .2))
  
  line((-2, 0), (2, 0), mark: (end: "barbed"))
  line((0, -2), (0, 2), mark: (end: "barbed"))

  line((0,1), (1,1), (1,0), stroke: (dash: "dashed"))

  line((0,0), (1,1), mark: (end: "barbed"), stroke: red)

  group({
    rotate(30deg)
    stroke(blue)
    line((-2, 0), (2, 0), mark: (end: "barbed"))
    line((0, -2), (0, 2), mark: (end: "barbed"))
  })
})
```

![grafik](https://github.com/cetz-package/cetz/assets/519002/dcfc9b7b-72ef-4fd3-879d-705463470938)

**Reply by psads-git** [reply]
What a nice solution! Thanks, @johannes-wolf!
